Makefile

```diff
@@ -41,7 +41,7 @@ deployKEI:
 	bash ./ci/kei-build-push.sh main
 
 deployKEIBeta:
-	bash ./ci/kei-build-push.sh beta
+	bash ./ci/kei-build-push.sh beta -beta
```

Comment:
I think you don't need this addition? I don't see a second argument being read by the script.